src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,256 @@
+import pinIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import NoBreachesIllustration from "../images/high-risk-breaches-none.svg";
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { SubscriberBreach } from "../../../../../../../../utils/subscriberBreaches";
+import { useL10n } from "../../../../../../../hooks/l10n";
+import { ResolutionContentLayout } from "../ResolutionContentLayout";
+
+type HighRiskBreachLayoutProps = {
+  typeOfBreach: "creditCard" | "ssnBreaches" | "bankAccount" | "pin" | "none";
+  breachData: GuidedExperienceBreaches;
+};
+
+export const HighRiskBreachLayout = (props: HighRiskBreachLayoutProps) => {
+  const l10n = useL10n();
+  const highRiskDataBreaches = props.breachData.highRisk;
+  let exposedData: SubscriberBreach[] = [];
+
+  if (props.breachData) {
+    switch (props.typeOfBreach) {
+      case "ssnBreaches":
+        exposedData = highRiskDataBreaches.ssnBreaches;
+        break;
+      case "creditCard":
+        exposedData = highRiskDataBreaches.creditCardBreaches;
+        break;
+      case "bankAccount":
+        exposedData = highRiskDataBreaches.bankBreaches;
+        break;
+      case "pin":
+        exposedData = highRiskDataBreaches.pinBreaches;
+        break;
+      default:
+        "none";
+        break;
+    }
+  }
+
+  // TODO: Make locale location-sensitive in the future
+  const dateFormatter = new Intl.DateTimeFormat("en-US", {
+    // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#datestyle
+    dateStyle: "short",
+  });
+
+  const listOfBreaches = exposedData.map((item: SubscriberBreach) => (
+    <div key={item.id} className={styles.breachItem}>
+      {/* TODO: Localize all date values MNTOR-2111 */}
+      {l10n.getFragment("high-risk-breach-name-and-date", {
+        elems: { breach_date: <span className={styles.date} /> },
+        vars: {
+          breach_name: item.name,
+          breach_date: dateFormatter.format(new Date(item.addedDate)),
+        },
+      })}
+    </div>
+  ));
+
+  const breachList = (
+    <div className={styles.breachItemsWrapper}>{listOfBreaches}</div>
+  );
+
+  let title, secondaryDescription, recommendationSteps, breachIllustration;
+
+  const CreditCardRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-three")}</li>
+    </ol>
+  );
+
+  const BankAccountRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-three")}</li>
+    </ol>
+  );
```

Comment:
Question: With even more content being added in the future would it make sense to move the content into a separate file?